### DISCUSSION TITLE
Make all hyperlinks ajaxified

### DISCUSSION THREAD

**Discussion by trta911** [jiripudil/Naja#298] [Support] [answered]
Can naja make by default all hyperlinks and forms ajaxified - without adding class "ajax" in the "a" tag?

Thank you.

**Comment by jiripudil** [comment, marked as answer]
Hello, yes, you can [change the selector](https://naja.js.org/#/ui-binding?id=custom-selector) to an empty string which will cause Naja to match all links and forms. You should do this prior to calling `naja.initialize()`.

```js
naja.uiHandler.selector = ''
```

Note that Naja only handles requests leading to an [allowed origin](https://naja.js.org/#/ui-binding?id=allowed-origins), others are still handled natively by the browser.

**Reply by trta911** [reply]
Thank you, it works as I expect :-) 

Is there also some option how to also make explicitly some of the hyperlinks not to be handled by ajax? Oposite functionality -> maybe by addingclass "noajax" or so.

**Reply by jiripudil** [reply]
Sure, Naja basically matches `` `a${uiHandler.selector}` ``, so you can do something like this:

```js
naja.uiHandler.selector = ':not(.noajax)'
```

It's all covered in the docs :)

**Reply by trta911** [reply]
Great, thank you for your patience, I am wrong docs reader :-)
